test/cli/remote-file/download-file-disk/expected-stderr

```diff
@@ -0,0 +1 @@
+%A downloading 100% %A
```

Comment:
Progress jde standardně do stderr. Imho je to proto, aby se dal sledovat progress i při pajpování.